protocol/app/module_accounts_test.go

```diff
@@ -34,6 +35,7 @@ func TestModuleAccountsToAddresses(t *testing.T) {
 		vestmoduletypes.CommunityTreasuryAccountName: "dydx15ztc7xy42tn2ukkc0qjthkucw9ac63pgp70urn",
 		vestmoduletypes.CommunityVesterAccountName:   "dydx1wxje320an3karyc6mjw4zghs300dmrjkwn7xtk",
 		icatypes.ModuleName:                          "dydx1vlthgax23ca9syk7xgaz347xmf4nunefw3cnv8",
+		wasmtypes.ModuleName:                         "dydx1xds4f0m87ajl3a6az6s2enhxrd0wta48eljwng",
```

Comment:
Usually there's no need to explicitly define a module address as a module account, which could lead to unexpected behavior when user send funds to it. Any specific reason here?